react/getting_started_with_react/what_is_jsx/what_is_jsx.md

```diff
@@ -0,0 +1,235 @@
+### Introduction
+
+You're likely somewhat familiar with JSX from previous React lessons, in this lesson we're going to explain _what_ it is, _why_ we use it, and _how_ to use it in your React apps.
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What JSX is.
+- Why we use JSX.
+- Differences between JSX and HTML.
+- How to write and take advantage of JSX's features.
+
+### What is JSX?
+
+JSX is a syntax extension for Javascript that lets you write HTML-like markup inside a Javascript file. It's not required to use JSX when writing React components, but it does make writing them more simple and concise.
+
+Essentially, JSX is syntactic sugar for the React [createElement](https://reactjs.org/docs/react-api.html#createelement) function. This function creates a React element, which is a plain object, so JSX compiles down to plain JavaScript objects.
+
+Here you can see the logged value of a div JSX element with some text:
+
+![JSX element's logged value](./imgs/00.png)
+
+### Why do we use JSX?
+
+Most of the time in apps, rendering logic and markup are inherently coupled, but we typically separate them by having our logic and markup in separate files. JSX allows React to separate concerns by containing both rendering logic and content in the same place (a component).
+
+On top of this, it's a more intuitive, visual way to work with the UI inside your code, and allows React to show more useful error and warning messages.
+
+### Converting HTML to JSX
+
+If you were to take some valid HTML and copy it straight into your React component, it would not work. This is due to some of the rules JSX implements, that aren't present in HTML.
+
+#### Rules of JSX:
```

Comment:
I think we should make Rules of JSX a first level heading, and move the "Converting HTML to JSX" heading after the rules section.

```suggestion
### Rules of JSX

If you were to take some valid HTML and copy it straight into your React component, it would not work. This is due to some of the rules JSX implements, that aren't present in HTML.
```